include/deal.II/base/vectorization.h

```diff
@@ -3120,7 +3120,7 @@ namespace std
     for (unsigned int i=0; i<dealii::VectorizedArray<Number>::n_array_elements; ++i)
       values[i] = std::sin(x[i]);
     ::dealii::VectorizedArray<Number> out;
-    out.load(&values[0]);
+    out.load(values);
```

Comment:
I have to admit that I have a preference for the former form if we can assume that `values` is not `nullptr`. That's because the syntax suggests that this is indeed an array of which we take the address of the first element. The latter is just a pointer, and so I have to look up whether that's a pointer to just one element or a pointer to an array.

I don't feel strongly about this, though, and will gladly leave it to others whether they want it this way or some other way.

Reply:
I agree with @bangerth .

Reply:
This is a good point: the current notation suggests that the function might expect a fixed-size array passed by reference instead of just a pointer. I'll fix this.